crates/bevy_core_pipeline/src/tonemapping/mod.rs

```diff
@@ -128,35 +227,107 @@ pub fn queue_view_tonemapping_pipelines(
     pipeline_cache: Res<PipelineCache>,
     mut pipelines: ResMut<SpecializedRenderPipelines<TonemappingPipeline>>,
     upscaling_pipeline: Res<TonemappingPipeline>,
-    view_targets: Query<(Entity, &Tonemapping)>,
+    view_targets: Query<(Entity, Option<&Tonemapping>, Option<&Dither>), With<ViewTarget>>,
 ) {
-    for (entity, tonemapping) in view_targets.iter() {
-        if let Tonemapping::Enabled { deband_dither } = tonemapping {
-            let key = TonemappingPipelineKey {
-                deband_dither: *deband_dither,
-            };
-            let pipeline = pipelines.specialize(&pipeline_cache, &upscaling_pipeline, key);
-
-            commands
-                .entity(entity)
-                .insert(ViewTonemappingPipeline(pipeline));
-        }
+    for (entity, tonemapping, dither) in view_targets.iter() {
+        let key = TonemappingPipelineKey {
+            deband_dither: *dither.unwrap_or(&Dither::Disabled),
+            tonemapping: *tonemapping.unwrap_or(&Tonemapping::None),
+        };
+        let pipeline = pipelines.specialize(&pipeline_cache, &upscaling_pipeline, key);
+
+        commands
+            .entity(entity)
+            .insert(ViewTonemappingPipeline(pipeline));
     }
 }
 
-#[derive(Component, Clone, Reflect, Default, ExtractComponent)]
+#[derive(
+    Component,
+    Debug,
+    Hash,
+    Clone,
+    Copy,
+    Reflect,
+    Default,
+    ExtractComponent,
+    PartialEq,
+    Eq,
+    FromReflect,
+)]
 #[extract_component_filter(With<Camera>)]
 #[reflect(Component)]
-pub enum Tonemapping {
+pub enum Dither {
```

Comment:
```suggestion
pub enum DebandDither {
```

Can I suggest `DebandDither`? Dithering can be used for a lot of things, this would disambiguate usage, and we did previously have it named `deband_dither`.

Reply:
I agree. DebandDither is better.